scripts/create_testenv.sh

```diff
@@ -23,6 +23,12 @@ if [[ $* != *--global* ]]; then
     source activate ${ENVNAME}
 fi
 
+if [ "$PYTHON_VERSION" = "3.5" ]; then
```

Comment:
Why only test in one version of python? Does pyro not work in 3.6?

Reply:
they're good questions! leaving answers here in addition to code for posterity!  this manually grabs the python 3.5 and 3.6 wheels. it is super brittle, but the only way I can think of doing it while there's a massive download on pypi (torch 1.0 was just released, so that might make it easier?)